config/nxf_call_str.config

```diff
@@ -28,6 +29,14 @@ params {
         variant_catalog = "${projectDir}/resources/GRCh38/expansionhunter_variant_catalog.json"
       }
     }
+    straglr{
+      minSupport = 1
+      minClusterSize = 1
+
+      GRCh38 {
+        loci = "${projectDir}/resources/GRCh38/clinical_repeats.bed"
```

Comment:
GRCh37: as discussed, we probably will never do long read on GRCh37, so its a waste of time to liftover the catalog. If people realy want this they can provide their own

Reply:
"the file contains only 38 loci, the expansion hunter catalog contains many more. I would expect them to be of equalish size. what do you think?"

odd indeed, however since this seeps to be the version epi2me is using as well I'm not sure where to find a better one...